packages/core/src/attestation/Attestation.ts

```diff
@@ -209,10 +237,25 @@ export default class Attestation implements IAttestation {
    *
    * @returns A promise containing the unsigned SubmittableExtrinsic (submittable transaction).
    * @example ```javascript
-   * // Use `store` to store an attestation on chain, and to create an `AttestedClaim` upon success:
-   * attestation.store().then(() => {
-   *   // the attestation store tx was successfully prepared, so now we can sign and send it and subsequently create an `AttestedClaim`.
+   * Attestation.store().then((creationExtrinsic) => {
+   *   // The attestation creation tx was created, and it can now be signed by a DID owner.
+   *   const attestationOwnerDid = did;
```

Comment:
C: This does not seem to be needed

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/974293be648c3fafc24709faf09638530614cb5c